### DISCUSSION TITLE
Environmental Sensors?

### DISCUSSION THREAD

**Discussion by Rochambeaux** [mcguirepr89/BirdNET-Pi#1097] [Q&A]
Is there a way to incorporate environmental sensor/data  to a BirdNet-Pi station?

**Comment by pddpauw** [comment]
no, or not that I'm aware of.  But you could very easily do this yourself, e.g. with a python script.
There are plenty of example on the web on how to do this.  You could then integrate those values in the homepage (adapting some of the php code).

You could store those in a separate DB, or -but a more difficult- add those values to recordings in the existing DB.

Good luck!
